### DISCUSSION TITLE
Сохранение изначальных значений в поле после авто заполнения

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#254] [Как это сделать?] [answered]
Здравствуйте, подскажите пожалуйста, как сделать чтобы в поле, которое автозаполняется кодом из таблицы пользователей сохранялось значение составившего документ изначально и того, кто потом редактировал? Сделал так: listCreate(item: $user)
user: nowUser() но значение меняется.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Немного продвинутая фигня — нужно использовать предыдущее значение поля и к нему прибавить текущего пользователя и почистить дубли:

```
= : listUniq(list: $add)

add: listAdd(list: $#onfv; item: $#nu)

```
